lib/mongoidable/class_abilities.rb

```diff
@@ -26,6 +26,13 @@ def inherits_from
         @inherits_from ||= superclass.respond_to?(:inherits_from) ? superclass.inherits_from.dup : []
       end
 
+      def accepts_policies_as(name)
```

Comment:
class method for defining policy relationships. Policy abilities are transposed through the relationships onto the parent model. 

Example
```
class User
  accepts_policies_as :organization_policies
end

u = User.new
u.organization_policies.create(requirements: {}, policy: {})
```